example/.ocamlformat

```diff
@@ -0,0 +1,10 @@
+version=0.18.0
```

Comment:
Can't we put a single `.ocamlformat` at the root of QCheck directory, rather than duplicating this file? According to https://github.com/ocaml-ppx/ocamlformat#options this should be enough, right?

Reply:
Done

Reply:
I don't have the permission to resolve my own threads, but now it's ok for me :+1: